python/python/Ice/_CommunicatorI.py

```diff
@@ -1,10 +1,10 @@
 # Copyright (c) ZeroC, Inc. All rights reserved.
 
 from .Communicator import Communicator
-from .PropertiesI import PropertiesI
-from .ObjectAdapterI import ObjectAdapterI
-from .ImplicitContextI import ImplicitContextI
-from .LoggerI import LoggerI
+from ._PropertiesI import PropertiesI
+from ._ObjectAdapterI import ObjectAdapterI
+from ._ImplicitContextI import ImplicitContextI
+from ._LoggerI import LoggerI
 from .Logger import Logger
 
 class CommunicatorI(Communicator):
```

Comment:
In many languages, we merged Communicator and CommunicatorI, Properties and PropertiesI (and more). Could we do the same in Python?

Reply:
We can, the only caveat is that the constructor takes a reference to the corresponding IcePy type. I guess this is fine if we keep it undocumented.

Reply:
Fixed in #2554